aiida/orm/importexport.py

```diff
@@ -381,6 +386,10 @@ def import_data_dj(in_path,ignore_unknown_nodes=False,
                                  "is neither a (possibly compressed) tar file, "
                                  "nor a zip file.")
 
+        if not folder.get_content_list():
+            print "The provided file/folder is empty. Exiting silently"
```

Comment:
"Exiting silently" while printing is a bit weird no? Would it be better to raise an exception and catch it in the caller, who can then decide what to print to the user.

Reply:
The main idea is to avoid a more "violent" exception that will occur later when the metadata.json will not be found at the folder. The silent exit makes sense when running import in a script and an empty import files, doesn't stop a full script of 100 imports to run.

I can of course raise an exception and catch it and exit silently at import_data or one level above but I don't see a lot of advantages. The only advantage that I see is that I can unify the behaviour of import_data_dj & import_data_sqla.

Reply:
Since these functions will be called mostly from `verdi` commands, we need to be able to control how the output is printed. Hard coding print statements in these functions makes that impossible. Just above this line, a `ValueError` is also raised to signal that the passed export file or folder is unusable. I don't see a reason why not to do the same here. But I notice that there are currently a load more print statements, so unless all of those are refactored, I guess changing just this one line won't accomplish too much